governance/xc_admin/packages/xc_admin_cli/package.json

```diff
@@ -16,7 +16,8 @@
   },
   "scripts": {
     "build": "tsc",
-    "format": "prettier --write \"src/**/*.ts\""
+    "format": "prettier --write \"src/**/*.ts\"",
+    "cli": "npm run build && node lib/cli.js"
```

Comment:
This command is a bit brittle as the build is not stanalone in the monorepo. Can you either run build using `npx lerna run build --scope="xc_admin_cli" --include-dependencies && ..`